pretty_printers_tests/Settings_macos.toml

```diff
@@ -0,0 +1,8 @@
+test_dir = "./tests"
+pretty_printers_path = "./../prettyPrinters/"
+lldb_batchmode = "./lldb_batchmode.py"
+lldb_lookup = "lookup"
+lldb_python = "/Applications/Xcode.app/Contents/SharedFrameworks/LLDB.framework/Resources/Python"
```

Comment:
Could you add `# BACKOMPAT: 2018.2` comment here to use python from clion distribution

Reply:
I mean to add `# BACKOMPAT: 2018.2` with some useful message to understand what need todo here